src/store/index.js

```diff
@@ -16,11 +16,7 @@ export function init (...middleware) {
     ]);
 
     registry.register({
-        compareReducer: reducers.compareReducer,
-        addSystemModalReducer: reducers.addSystemModalReducer,
-        errorAlertReducer: reducers.errorAlertReducer,
-        filterDropdownReducer: reducers.filterDropdownReducer,
-        actionKebabReducer: reducers.actionKebabReducer
+        appState: appReducer
```

Comment:
Do I understand this correctly that all of the reducers now live within a single slice of the state known as `appState` ?